ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/support/commands.js

```diff
@@ -224,4 +224,16 @@ Cypress.Commands.add('disMiles', () => {
 // Distance Info (General Info Page) - Save & Continue Button ID
 Cypress.Commands.add('saveContinue', () => {
     cy.get('[id="save-and-continue-button"]')
+})
+
+// Request external dev
+Cypress.Commands.add('beData', () => {
+    cy.request({
+        method:'GET',
+        url:'https://trams-external-api.azurewebsites.net/v2/apply-to-become/application/A2B_1373',
```

Comment:
Is this ok to be hardcoded?

Reply:
It is hardcoded at the moment, not sure how I can get around this. Any suggestions?

Reply:
Honestly I'm not sure, but maybe we need to have a way of choosing a url based on dev/pre-prod. In terms of the actual application ref, if it's going to permanently exist I think it's fine